test/edge-cases.test.ts

```diff
@@ -71,17 +71,17 @@ describe("edge cases", () => {
     });
   });
 
-  context("try to decode invalid MessagePack binary", () => {
-    it("throws errors", () => {
-      const decoder = new Decoder();
-      const TYPE_NEVER_USED = 0xc1;
-
-      assert.throws(() => {
-        decoder.decode([TYPE_NEVER_USED]);
-      }, /unrecognized type byte/i);
-      testDecoder(decoder);
-    });
-  });
+  // context("try to decode invalid MessagePack binary", () => {
+  //   it("throws errors", () => {
+  //     const decoder = new Decoder();
+  //     const TYPE_NEVER_USED = 0xc1;
```

Comment:
Does this work if you just change it to `0xc4`? If so, that'd be cleaner.

Otherwise, I'm good with commenting it out.

Reply:
No, once we add 0xc1, this closes the entire usable range from 0xc0 to 0xe0 (I checked every one 🙂)

Reply:
Makes sense. Should we do `it.skip(...)` to make it clear that the test is skipped and add a comment why?